src/main/java/org/aya/intellij/parser/AyaParserDefinition.java

```diff
@@ -42,26 +32,6 @@ public class AyaParserDefinition extends SyntaxHighlighterFactory implements Par
     return new AyaPsiParser();
   }
 
-  /**
-   * Create the root of your PSI tree (a PsiFile).
-   * <p>
-   * From IntelliJ IDEA Architectural Overview:
-   * "A PSI (Program Structure Interface) file is the root of a structure
-   * representing the contents of a file as a hierarchy of elements
-   * in a particular programming language."
-   * <p>
-   * PsiFile is to be distinguished from a FileASTNode, which is a parse
-   * tree node that eventually becomes a PsiFile. From PsiFile, we can get
-   * it back via: {@link PsiFile#getNode()}
-   */
-  @Override public @NotNull PsiFile createFile(@NotNull FileViewProvider viewProvider) {
-    return new AyaPsiFile(viewProvider);
-  }
-
-  @Override public @NotNull PsiElement createElement(@NotNull ASTNode node) {
-    return AyaPsiElementTypes.Factory.createElement(node);
-  }
-
```

Comment:
Why delete these? They are required by IJ

Reply:
Oh my god I didn't mean to I used copy and paste w/o realizing the differences

Reply:
LOL

Reply:
We should decouple ParserDefinition and use the minimal one in upstream and the extended one in IJ plugin.